rest/config_test.go

```diff
@@ -887,3 +888,41 @@ func TestPutInvalidConfig(t *testing.T) {
 	response := rt.SendAdminRequest("PUT", "/db/_config", `{"db": {"server": "walrus"}}`)
 	assert.Equal(t, http.StatusBadRequest, response.Code)
 }
+
+// Validate basic mapping from config to database options
+func TestConfigToDatabaseOptions(t *testing.T) {
```

Comment:
It feels like we should have a mechanism to test config option mappings in a more general or reusable way, but out of scope for this PR.

Reply:
I agree - I was looking for a test that already did this and didn't find anything - my thinking was that we could expand this test in future.